integration-cli/daemon/daemon_unix.go

```diff
@@ -18,7 +18,7 @@ func cleanupExecRoot(c *check.C, execRoot string) {
 	// new exec root.
 	netnsPath := filepath.Join(execRoot, "netns")
 	filepath.Walk(netnsPath, func(path string, info os.FileInfo, err error) error {
-		if err := unix.Unmount(path, unix.MNT_FORCE); err != nil {
+		if err := unix.Unmount(path, unix.MNT_FORCE); err != nil && err != unix.EINVAL {
```

Comment:
Should we also ignore `ENOENT`? 

```
04:02:02 unmount of /tmp/docker-execroot/d1825ad513bf4/netns failed: invalid argument
04:02:02 unmount of /tmp/docker-execroot/d1825ad513bf4/netns failed: no such file or directory
```

Reply:
Yes, I think ENOENT can be returned due to races and should be ignored.

Oh, and `filepath.Walk()` should not be used here as it's recursive and does lots of extra things we don't need. Using `os.Readdirnames()` should be sufficient (similar to https://github.com/moby/moby/pull/36307).